tests/envs/test_env_implementation.py

```diff
@@ -126,6 +126,16 @@ def test_frozenlake_dfs_map_generation(map_size: int):
     raise AssertionError("No path through the frozenlake was found.")
 
 
+@pytest.mark.parametrize("map_size, seed", [(5, 123), (10, 42), (16, 987)])
+def test_frozenlake_map_generation_with_seed(map_size: int, seed: int):
+    map1 = generate_random_map(size=map_size, seed=seed)
+    map2 = generate_random_map(size=map_size, seed=seed)
+    assert map1 == map2
+    map1 = generate_random_map(size=map_size)
```

Comment:
I would add this as there are cases where this could fail. 
You could replace it with a test that `seed` and `seed+1` have different maps

Reply:
Indeed... updated as you suggested.